src/printer/color.rs

```diff
@@ -4,16 +4,18 @@ use std::{error, fmt, str::FromStr};
 
 use termcolor::{Color, ColorSpec, ParseColorError};
 
-#[allow(dead_code)]
 pub fn default_color_specs() -> Vec<UserColorSpec> {
     vec![
         #[cfg(unix)]
-        "path:fg:magenta".parse().unwrap(),
+        "path:fg:green".parse().unwrap(),
```

Comment:
I forgot that I had `ripgrep`'s default colors overridden in a local config file

I can't with the default `ripgrep` colors they're disgusting

So this is "my" color scheme (don't remember where I got it from or if I made it up)

Reply:
I find the highlights pretty gross when running in my terminal using my color scheme (I assume this is somehow using the colors I have specified there?):

<img width="673" alt="Screenshot 2023-07-20 at 5 12 37 PM" src="https://github.com/helixbass/tree-sitter-grep/assets/98044/8bacf6bd-2ca3-401d-b7bd-4d9d399d7eb9">

But it's readable 🤷🏻

Reply:
Ha yes, 100% of people want it to be configurable, I guess, and I definitely couldn't be bothered to pass `--colors` every time I used it.